rbs/prism/CommentsAssociatorPrism.cc

```diff
@@ -537,11 +537,22 @@ void CommentsAssociatorPrism::walkNode(pm_node_t *node) {
 
             associateAssertionCommentsToNode(node);
 
-            walkNode(block->body);
+            block->body = walkBody(node, block->body);
```

Comment:
Do we need to free the old `block->body`? I assume not, because we're just rearranging the tree in-place, and not returning a full modified/copied tree.

Reply:
nvm, even single expression lambda bodies still get wrapped into a `pm_statements_node`